core/rend/vulkan/vulkan_context.cpp

```diff
@@ -728,6 +728,18 @@ bool VulkanContext::init()
 	float hdpi, vdpi;
 	if (!SDL_GetDisplayDPI(SDL_GetWindowDisplayIndex(sdlWin), nullptr, &hdpi, &vdpi))
 		settings.display.dpi = roundf(std::max(hdpi, vdpi));
+
+#ifdef __linux__
+	// Fixing Steam Deck's incorrect 60mm * 60mm EDID 
+	if (settings.display.dpi > 500)
+	{
+		int displayIndex = SDL_GetWindowDisplayIndex(sdlWin);
+		SDL_DisplayMode mode;
+		SDL_GetDisplayMode(displayIndex, 0, &mode);
+		if ( displayIndex == 0 && strcmp(SDL_GetDisplayName(displayIndex), "ANX7530 U") && mode.w == 1280 && mode.h == 800 )
```

Comment:
Just checking here: you don't want to change the DPI for "ANX7530 U" screens?

Reply:
thanks, after double checking the dimension is printed into the display name for X11, so the correct value is `ANX7530 U 3"`, fixed the strcmp also

Reply:
So screens named `ANX7530 U 3"` should not have their DPI changed?
Sorry for insisting but `strcmp()` returns true if strings are different, which is often source of confusion

Reply:
When 
- DPI > 500
- is primary display
- display name = `ANX7530 U 3"`
-  width = 1280
-  height = 800

we will hard patch the dpi to 206

the `strcmp()` check is updated to `strcmp(SDL_GetDisplayName(displayIndex), "ANX7530 U 3\"") == 0` now
the previous success is by double negative when using the wrong name